### PR TITLE
fixed a typo

### PR DESCRIPTION
**Pull request by badersur**
The sentence:

> If you need to get Ubuntu, check out the Ubuntu Downloads website.

looks odd to me. I think if someone reached here then he should know about Ubuntu or at least know how to obtain it! (maybe adding a link to the download page won't make it too odd!)

### CODE REVIEW

File: _includes/partials/install_sql_server_linux_ubuntu.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Note: To ensure optimal performance of SQL Server, your machine should have at least 4 GB of memory.
 If you need to get Ubuntu, check out the Ubuntu Downloads website.
 
-1. Register the Microsoft Linux repositories and add they keys
+1. Register the Microsoft Linux repositories and add their keys
```
Comment (badersur) [inline _includes/partials/install_sql_server_linux_ubuntu.md]: Maybe should use **these** instead of **their**?!

Comment (ajlam) [inline _includes/partials/install_sql_server_linux_ubuntu.md]: Yes, you are definitely correct 😄

**Comment by ajlam** [comment]
Thanks for your PR 👍

**Comment by badersur** [comment]
You are welcome @ajlam :)

**Pull request merged**